examples/nlp/intent_detection_slot_tagging/joint_intent_slot_infer.py

```diff
@@ -34,9 +34,8 @@
 parser.add_argument("--batch_size", default=128, type=int)
 parser.add_argument("--max_seq_length", default=50, type=int)
 parser.add_argument("--pretrained_bert_model", default="bert-base-uncased", type=str)
-parser.add_argument("--dataset_name", default='snips-all', type=str)
-parser.add_argument("--data_dir", default='data/nlu/snips', type=str)
-parser.add_argument("--work_dir", required=True, help="your checkpoint folder", type=str)
+parser.add_argument("--data_dir", default='data/atis', type=str)
```

Comment:
If I just remove the common, it can not find it, but if I remove trainables.common, it is ok. So as we need to keep huggingface, there is no other way. Do you know how we can fix it? I remember that we tried and couldn't.

Reply:
What about doing one of the following options: 1.`from nemo.collections.nlp.nm.trainables import huggingface` and then `model  = huggingface.BERT()' or 2. import nemo.collections.nlp as nemo_collections and then `model = nemo_nlp.nm.trainables.huggingface.BERT()`

Reply:
Used get_huggingface_model so no need to import huggingface directly. But it seems that the second option does not work. It can not access huggingface from trainables directly. It is weird!

Reply:
weird, both options work for me